File: module_login/src/main/java/com/yyxnb/module_login/ui/LoginActivity.java
package com.yyxnb.module_login.ui;

import android.support.v4.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.common_base.core.ContainerActivity;

import static com.yyxnb.common_res.arouter.ARouterConstant.LOGIN_MAIN;

@Route(path = LOGIN_MAIN)
public class LoginActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new LoginFragment();
    }
}

File: module_login/src/main/java/com/yyxnb/module_login/ui/LoginActivity.kt
package com.yyxnb.module_login.ui

import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.yyxnb.common_base.arouter.ARouterConstant.LOGIN_ACTIVITY
import com.yyxnb.common_base.base.ContainerActivity

@Route(path = LOGIN_ACTIVITY)
class LoginActivity : ContainerActivity() {

    override fun initBaseFragment(): Fragment? {
        return LoginFragment()
    }
}